### PR TITLE
[4.0] com_joomlaupdate typos

### PR DESCRIPTION
**Pull request by brianteeman**
code review

### CODE REVIEW

File: administrator/components/com_joomlaupdate/restore.php [hunk 1]
```diff
@@ -3111,7 +3111,7 @@ class AKText extends AKAbstractObject
 		'BTN_SITEFE'                      => 'Visit your site\'s frontend',
 		'BTN_SITEBE'                      => 'Visit your site\'s backend',
 		'WARNINGS'                        => 'Extraction Warnings',
-		'ERROR_OCCURED'                   => 'An error occurred',
+		'ERROR_OCCURRED'                   => 'An error occurred',
```
Comment (Quy) [inline administrator/components/com_joomlaupdate/restore.php]:
```suggestion
		'ERROR_OCCURRED'                  => 'An error occurred',
```

**Comment by ChristineWk** [comment]
Prebuild Package used. In principle sucessful, except that I don't know how to check the file:

build/media_source/com_joomlaupdate/js/default.es6.js

**Comment by brianteeman** [comment]
You can check the generated file
media\com_joomlaupdate\js\default.js

**Comment by ChristineWk** [comment]
> 
> 
> You can check the generated file
> media\com_joomlaupdate\js\default.js

I had also checked the above before. Checked now again:

Line 367:
`// eslint-disable-next-line max-len`

Line 378:
`html = Joomla.JText._('COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSION_NO_COMPATIBILITY_INFORMATION');`

Line 397:
`document.getElementById('compatibilitytype0').style.display = 'block'; // Process the nonCoreCriticalPlugin list`
Line 398: blank

**Comment by brianteeman** [comment]
The line numbers will be different just look at the changed lines and find and compare
https://github.com/joomla/joomla-cms/pull/34456/files

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on cfedaf8a82beda299a03b82aef50cf1229ebc046<br /><br />code changes look good.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34456">issues.joomla.org/tracker/joomla-cms/34456</a>.</sub>

**Comment by RickR2H** [comment]
I have tested this item :white_check_mark: successfully on cfedaf8a82beda299a03b82aef50cf1229ebc046<br /><br />Look good!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34456">issues.joomla.org/tracker/joomla-cms/34456</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34456">issues.joomla.org/tracker/joomla-cms/34456</a>.</sub>

**Comment by chmst** [comment]
Thanks!

**Comment by brianteeman** [comment]
thanks

**Pull request merged**